README.md

```diff
@@ -1,8 +1,54 @@
-# Imagery Processing 
+# Topo Processor 
 
 [![GitHub Actions Status](https://github.com/linz/topo-processor/workflows/Build/badge.svg)](https://github.com/linz/topo-processor/actions)
 [![Alerts](https://badgen.net/lgtm/alerts/g/linz/topo-processor?icon=lgtm&labelColor=2e3a44&label=Alerts&color=3dc64b)](https://lgtm.com/projects/g/linz/topo-processor/context:python)
 [![Dependabot Status](https://badgen.net/dependabot/linz/topo-processor?icon=dependabot&labelColor=2e3a44&color=blue)](https://dependabot.com)
 [![License](https://badgen.net/github/license/linz/processor-aerial-imagery?labelColor=2e3a44&label=License)](https://github.com/linz/topo-processor/blob/master/LICENSE)
 [![Conventional Commits](https://badgen.net/badge/Commits/conventional?labelColor=2e3a44&color=EC5772)](https://conventionalcommits.org)
 [![Code Style](https://badgen.net/badge/Code%20Style/black?labelColor=2e3a44&color=000000)](https://github.com/psf/black)
+
+## Description
+The topo processor is a collection of small components that can be combined together to create a pipeline. 
+
+The pipeline can have many ways of being run (CLI vs webservice) as well as many ways of being used (uploading data vs validating vs transforming)
+
+The first pipeline being created is a upload validation pipeline for Historical Imagery.
+## Installation
+#### Requirements:
+**Poetry:**
+Follow the [Poetry installation guide](https://python-poetry.org/docs/)
+**Docker:**
+follow the Installation guide [(Ubuntu Installation)](https://docs.docker.com/engine/install/ubuntu/)
+
+#### Recommended:
+- [node](https://nodejs.org/en/about/)
+- [Pretty JSON Logger](https://npmjs.com/package/pretty-json-log)
+
+#### Use poetry to install 
+
+```shell
+poetry shell
+
+poetry install
+```
+## Usage
```

Comment:
It is currently describing only the usage of the `upload` functionality but should have more entry points later. It could be an idea to add a sub section 'Upload'